src/containers/Counter.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+// components
+import TitleValue from '../components/TitleValue';
+/* TODO:
+  1. Refactor this to use function,
+  2. Move the logic for dispatch and state to props into a mapping for the component created under containers.
+*/
+class Counter extends Component {
+  render() {
+    return (<div>
+    <TitleValue label={this.props.label} initialValue={this.props.number} /><br/>
+    <button onClick={this.props.onIncrement}> Increment </button>
+    {/* Adding empty span's to create space between the buttons. */}
+    <span></span> <span></span>
```

Comment:
This is alright for now, but we don't want to add HTML elements for the sake of accomplishing spacing. Ideally this would use CSS, but we will get into that when go through styled components.

Reply:
Yes, i was not sure where to add CSS. Hence, i added the comment as well.